### PR TITLE
Add node version filtering for sampling

### PR DESCRIPTION
**Pull request by vzotova**
**Type of PR:**
- [ ] Bugfix
- [X] Feature
- [ ] Documentation
- [ ] Other

**Required reviews:** 
- [ ] 1
- [ ] 2
- [X] 3

**What this does:**
> High-level idea of the changes introduced in this PR. 
> List relevant API changes (if any), as well as related PRs and issues.

**Issues fixed/closed:**
> - Fixes #...

Closes https://github.com/nucypher/sprints/issues/36.

**Why it's needed:**
> Explain how this PR fits in the greater context of the NuCypher Network.
> E.g., if this PR addresses a `nucypher/productdev` issue, let reviewers know!

**Notes for reviewers:**
> What should reviewers focus on? 
> Is there a particular commit/function/section of your PR that requires more attention from reviewers?

### CODE REVIEW

File: porter/fields/base.py [hunk 1]
```diff
@@ -108,3 +109,12 @@ def _deserialize(self, value, attr, data, **kwargs):
                     f"Unexpected object type, {type(result)}; expected {self.expected_type}")
 
             return result
+
+
+class VersionString(String):
+
+    def _validate(self, value):
+        try:
+            parse(value)
```
Comment (derekpierre) [inline porter/fields/base.py:118]: Nice!

File: porter/schema.py [hunk 2]
```diff
@@ -125,6 +126,18 @@ class GetUrsulas(BaseSchema):
         ),
     )
 
+    min_version = VersionString(
```
Comment (derekpierre) [inline porter/schema.py:129]: We need to update the README to add this optional parameter.

Filed #75 which can be addressed in a separate PR.

**Review by derekpierre — APPROVED** [review]
🎸

File: porter/schema.py (hunk 2 above)
Comment (vzotova) [reply porter/schema.py:129]: Added small note in README for get_ursulas

**Review by KPrasch — APPROVED** [review]

**Pull request merged**